AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -118,6 +118,8 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+overclockers.co.uk#$#body { overflow-y: auto !important; }
+overclockers.co.uk#$#.modal { display: none !important; }
```

Comment:
Too wide, and scroll unlocking causes vertical empty space.
Added https://github.com/AdguardTeam/AdguardFilters/commit/780254deb46d2ea1a42c9dc9af8d554de6868534